### DISCUSSION TITLE
Deploying on Google Cloud

### DISCUSSION THREAD

**Discussion by dvaryani-bit** [reflex-dev/reflex#3164] [Q&A] [answered]
Is it possible to get the complied code of a Reflex app in Fast API and React, so I can deploy on Google?

**Comment by masenf** [comment, marked as answer]
`reflex export` should provide a `frontend.zip` which can be hosted with any static webserver, and a `backend.zip` which contains the python code that can be executed via `reflex run --backend-only`.

The only limitation with this approach is that you must configure missing pages to redirect to `/404` so that dynamic routes (which don 't have actual html files) can be properly loaded on the client side.

**Reply by dvaryani-bit** [reply]
thank you @masenf
